bin/completedUserActions.ts

```diff
@@ -0,0 +1,97 @@
+import '../src/config';
+import {
+  Alerts,
+  Comment,
+  Post,
+  PostType,
+  SourceMember,
+  UserActionType,
+  User,
+} from '../src/entity';
+import createOrGetConnection from '../src/db';
+import { SourceMemberRoles } from '../src/roles';
+import { DataSource } from 'typeorm';
+
+const insertSelectAction = (
+  con: DataSource,
+  [query, params]: [string, unknown[]],
+) => con.query(`INSERT INTO user_action("userId", type) ${query}`, params);
```

Comment:
I think we should ignore conflicts right?

Reply:
Indeed we should 🚀